Yoast/javascript
# Add passive voice assessment for Turkish

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the passive voice assessment for Turkish.
* [Yoast SEO Free] Adds the assessment for detecting passive voice in Turkish.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Turkish.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`Çanta trende unutuldu.`
`Durumum beğenilmiş.`
`Bu sabah pencere açıldı.`
`Suç hakim tarafından affedilecek.`
`Görüşleri yargıç tarafından dinleniyor.`
`Sorun servis tarafından en kısa sürede çözülebilir.`
`Birçok girişimde bulunuldu.`
`Argümanlar başlatılabilir.`
`Öneri komisyona gönderildi.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add this sentence and check that it does not get highlighted:
`Geçen haftanın planını tartışıyoruz.`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=LINGO&modal=detail&selectedIssue=LINGO-540
